src/App.js

```diff
@@ -3,17 +3,27 @@ Redirects to different pages
 @returns home page
 @author:Shravya P
```

Comment:
Add details about what this component is all about say Entry of our application or Root Component.